src/libs/actions/SignInRedirect.js

```diff
@@ -16,12 +16,19 @@ Onyx.connect({
     callback: val => currentPreferredLocale = val,
 });
 
+let currentIsOffline;
```

Comment:
Can we just use `isOffline`?

Reply:
Yes, and related to the comment below we need it so that we can preserve the old state of `isOffline` so that we're avoiding it getting reset when Onyx.clear is called